src/main/kotlin/me/zeroeightsix/kami/feature/module/MurderMysteries.kt

```diff
@@ -0,0 +1,124 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting.Constrain
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.TickEvent
+import me.zeroeightsix.kami.util.Texts
+import net.minecraft.item.Items
+import net.minecraft.text.MutableText
+import net.minecraft.util.Formatting
+
+@Module.Info(
+        name = "MurderMysteries",
+        category = Module.Category.MISC,
+        description = "Get notified when a player holds a dangerous item"
+)
+object MurderMysteries : Module() {
+
+    @Setting(name = "InnocuousItems")
+    private var innocuousItems = true
+    @Setting(name = "WeaponItems")
+    private var weaponItems = true
+    @Setting(name = "ToolItems")
+    private var toolItems = true
+    @Setting(name = "DetectiveItems")
+    private var detectiveItems = false
+
+    @Setting(name = "CooldownOnOutput") // This is how often it will tell the player in chat
+    private var updateEvery: @Constrain.Range(min = 2.0, max = 16.0, step = Double.MIN_VALUE) Int = 16
+
+    private var lastUpdate: Long = 0L
+
+    private fun murdererIs(username: String, item: String): MutableText? {
+        lastUpdate = System.currentTimeMillis()
+        return Texts.f(Formatting.WHITE, Texts.append(
+                Texts.lit("[MurderMysteries] "),
+                Texts.flit(Formatting.AQUA, username),
+                Texts.flit(Formatting.WHITE, " is the "),
+                Texts.flit(Formatting.RED, "murderer! "),
+                Texts.flit(Formatting.WHITE, "Wields "),
+                Texts.flit(Formatting.GOLD, "$item!")
+        ))
+    }
+    private fun detectiveIs(username: String, item: String): MutableText? {
+        lastUpdate = System.currentTimeMillis()
+        return Texts.f(Formatting.WHITE, Texts.append(
+                Texts.lit("[MurderMysteries] "),
+                Texts.flit(Formatting.AQUA, username),
+                Texts.flit(Formatting.WHITE, " has "),
+                Texts.flit(Formatting.GOLD, "$item!")
+        ))
```

Comment:
I just introduced a new DSL for composing `Text`s, please merge `fabric` into your PR and switch to the new DSL. Look at commands for examples.